pkgs/unified_analytics/example/sample_rate.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:unified_analytics/src/utils.dart';
+
+void main() {
+  // Seed has been set to replicate results
+  //
+  // Test with your own seed and alter other parameters
+  // as needed
+  final uuidGenerator = Uuid(123);
```

Comment:
This is actually not for the tests, it's meant to be repeatable because I include the `stdout` when running the script to demonstrate that the sampling rate is working as expected

Example of the run below
```
// Output from run above with set parameters
//
// '''
// Test sample rate = 0.3
// Number of iterations = 10000
// ---

// Count of iterations sampled (successes) = 3046
// Actual sample rate = 0.3046
// ---

// Runtime = 8ms
// '''
```

Reply:
I have added documentation to the top of the example file that should make it clearer

Reply:
I would assume example code would be copy pasted, and best practices.